client/app/lib/accessibility.ts

```diff
@@ -0,0 +1,21 @@
+export function srNotify(text: string, expiry = 1000, container = document.body): void {
+  const element = document.createElement("div");
+  const id = `speak-${Date.now()}`;
+
+  Object.assign(element, {
+    id,
+    className: "sr-only",
+    textContent: text,
+  });
+
+  [
+    ["role", "alert"],
+    ["aria-live", "assertive"],
```

Comment:
Maybe `polite` instead? If they go in and out of the editor a lot, this could get tiresome. Or maybe assertiveness should be a parameter?

Reply:
I was with the mind too much in the specific case I was addressing, indeed this should be a parameter and its default value should be polite.